src/util/misc/vectors.ts

```diff
@@ -75,8 +73,6 @@ export const getBisector = (A: Point, B: Point, C: Point) => {
  */
 export const getOrthogonalUnitVector = (
   v: Point,
-  counterClockwise: boolean = true
```

Comment:
Just out of curiosity, does the typing cause any problems in this case?

Reply:
On the contrary
It marked it as redundant